### ISSUE TITLE
No compatible version found: broccoli-sass-source-maps@'>=0.7.0 <0.8.0'

### ISSUE THREAD

**Issue by chrismou** [aexmachina/ember-cli-sass#45]
An app I'm working on is attempting to install broccoli-sass-source-maps, seemingly as a dependency of ember-cli-sass 3.3.0 (it's referencing "^3.2.2" in package.json), and the install is failing with the following error:

```
npm ERR! notarget No compatible version found: broccoli-sass-source-maps@'>=0.7.0 <0.8.0'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.6.2","0.6.3","1.0.0-beta.1","1.0.0-beta.4"]
```

I tried this initially on a brand new DigitalOcean droplet, so I don't think caching is an issue. Have also tried the same on my local machine, and am seeing the same error.

I'm fairly new to node, so I'm unsure if this is just an oversight in publishing a package to npm, or if that's even a thing.  Let me know if you need any more info from me.

**Comment by daanwissing** [comment]
I just had the same issue a few minutes ago. Explicitly installing version 4.0.0-beta.4 (`npm install ember-cli-sass@4.0.0-beta.4`) worked for me.

**Comment by ZebraFlesh** [comment]
This should now be fixed (see #44) thanks to @aexmachina.  Just try `npm install` or `npm install ember-cli-sass@3.3.0` again.
